config/site/navigation.config.js

```diff
@@ -17,11 +18,16 @@ const navConfig = {
       path: '/components',
       text: 'Components',
       pageTypes: ['doc'],
+      capabilities: {
+        devToolbar: true,
```

Comment:
based on [my question](https://github.com/cerner/terra-dev-site/pull/175#issuecomment-515479335) and [your response](https://github.com/cerner/terra-dev-site/pull/175#issuecomment-515480135) should we rename to something like: `devTools`

Reply:
`devTools` sounds fine, I struggled with naming this guy.